optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/util/PlannerTestUtils.java

```diff
@@ -98,7 +98,7 @@
                 solverConfig.getScoreDirectorFactoryConfig();
         scoreDirectorFactoryConfig.setEasyScoreCalculatorClass(null);
         scoreDirectorFactoryConfig.setScoreDrlList(Collections.singletonList(
-                "org/optaplanner/core/impl/score/dummySimpleScoreDroolsConstraints.drl"));
+                "org/optaplanner/core/impl/score/director/dummyDroolsConstraints.drl"));
```

Comment:
I'd argue that now that drl actually has a constraint, it's no longer a dummy. Not sure if we want to leak that constraint to all test cases that use this method already.

I'd argue that the test case that really needs that "First weight" constraint in there, should add it explicitly